### DISCUSSION TITLE
关于TTL在配置中的位置问题

### DISCUSSION THREAD

**Discussion by pmkol** [IrineSistiana/mosdns#392] [Q&A]
当在前面定义过ttl插件的tag后，后续在分流规则逻辑中使用时，标签插入的位置有顺序要求么

例如：
        - if: query_is_local_domain
          exec:
            - _prefer_ipv4
            - ecs_cn
            - _single_flight
            - forward_local
            - ttl_cn
            - _return

forward_local 与 ttl_cn 标签的加载顺序是否有要求？

**Comment by IrineSistiana** [comment]
sequence里

forward 会获取应答。

ttl 会修改应答 ttl。

所以 ttl 必须在 forward 后面(不一定要紧挨着)。

---

和插件加载顺序无关。
